### ISSUE TITLE
questions about contrastive-caption-traning details

### ISSUE THREAD

**Issue by happywinder** [microsoft/LLM2CLIP#24]
In contrastive caption fine-tuning ,how do you train on CC-3M?Does a contrastive learning on modified CC-3M(with help of shareCaptioner, the dense caption and original caption is regarded  as positive)?

**Comment by Yif-Yang** [comment]
Yes, we used the enhanced CC3M dataset from ShareCaptioner, training it with one dense caption paired with the original caption.
